Automattic/wp-calypso
# Launchpad - Add Free flow `Choose a domain` task for users on free plan

Related to #72874

## Test / Review estimation

Test: short
Review: short

## Proposed Changes

* Add Launchpad free flow `domain_upsell` task to keep users engaged with upsell opportunities.
* `Choose a domain (Upgrade plan)` checklist task will navigate user to `domains/add` page

<img width="358" alt="CleanShot 2023-02-14 at 15 03 33@2x" src="https://user-images.githubusercontent.com/10482592/218848973-833d48d3-2c8e-4050-aa91-a2b5298fc34c.png">

## Testing Instructions



* Checkout this branch
* Create or use an existing Launchpad free flow site (on free plan).
* Navigate to the launchpad and verify the `Choose a domain (Upgrade plan)` task appears on the launchpad checklist.
* Click on `Choose a domain (Upgrade plan)` task and verify it navigates to `/domains/add/{site_slug}`
* Upgrade the site plan and navigate back to the launchpad screen.
* Verify the `Choose a domain (Upgrade plan` task does not appear on the launchpad checklist.

## Pre-merge Checklist
